src/StateSmith/Runner/SmRunner.cs

```diff
@@ -147,6 +149,23 @@ private void SetupDependencyInjectionAndRenderConfigs()
     {
         var renderConfigAllVars = new RenderConfigAllVars();
 
+        // if there's a toml file with the same diagram name, read its settings
+        string tomlPath = Path.ChangeExtension(this.settings.DiagramPath, ".toml");
+        if (File.Exists(tomlPath))
+        {
+            // TODO it would be nice to write "Reading toml settings from: {tomlPath}" to console
+            // but that doesn't seem to be a pattern in SmRunner. SmRunnerInternal has an
+            // IConsolePrinter, but SmRunner doesn't. Should I just grab one from DI?
+            try
+            {
+                new TomlReader(renderConfigAllVars, settings).Read(File.ReadAllText(tomlPath));
+            }
+            catch (IOException e)
```

Comment:
should probably also catch `TomlException` (toml lib throws) and `ArgumentException` (statesmith throws)